<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changeInfo</class>
 <widget class="QDialog" name="changeInfo">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#changeInfo{background-color: #cbd6da;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <layout class="QFormLayout" name="formLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;Yu Gothic UI Light&quot;;</string>
         </property>
         <property name="text">
          <string>Ім'я</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="name2Label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;Yu Gothic UI Light&quot;;</string>
         </property>
         <property name="text">
          <string>Прізвище</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="nameEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;Yu Gothic UI Light&quot;;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="name2Edit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;Yu Gothic UI Light&quot;;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="AgeLabel">
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;Yu Gothic UI Light&quot;;</string>
         </property>
         <property name="text">
          <string>Вік</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="AgeEdit">
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;Yu Gothic UI Light&quot;;</string>
         </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>12</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="gradeLabel">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;Yu Gothic UI Light&quot;;</string>
         </property>
         <property name="text">
          <string>Клас</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="gradeEdit">
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;Yu Gothic UI Light&quot;;</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="teacherLabel">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 15pt &quot;Yu Gothic UI Light&quot;;</string>
           </property>
           <property name="text">
            <string>Вчитель</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="teacherBox">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#teacherBox{
	font: 15pt &quot;Yu Gothic UI Light&quot;;
	background-color: rgb(255, 255, 255);
}
#teacherBox:hover{
	background-color: rgb(250, 191, 35);
}</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="7" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="passEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;Yu Gothic UI Light&quot;;</string>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Старий пароль</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="newPassEdit">
         <property name="styleSheet">
          <string notr="true">font: 15pt &quot;Yu Gothic UI Light&quot;;</string>
         </property>
         <property name="inputMask">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Новий пароль</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="infoLabel">
       <property name="styleSheet">
        <string notr="true">font:40pt &quot;Yu Gothic UI Light&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>ЗМІНА 
ОСОБИСТИХ ДАНИХ</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="passLabel">
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;Yu Gothic UI Light&quot;;
</string>
       </property>
       <property name="text">
        <string>ЗМІНА ПАРОЛЮ</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="2">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="numberLabel">
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;Yu Gothic UI Light&quot;;</string>
       </property>
       <property name="text">
        <string>Номер телефону:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="numberEdit">
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;Yu Gothic UI Light&quot;;</string>
       </property>
       <property name="text">
        <string>+380</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="readyButton">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#readyButton {
background-color:rgb(250, 191, 35);
color: rgb(45, 44, 44);
font: 20pt &quot;Yu Gothic UI Semilight&quot;;
border: 0px;}

#readyButton:hover{
background-color: rgb(45, 44, 44);
color: white;
font:20pt &quot;Yu Gothic UI Semilight&quot;;}</string>
       </property>
       <property name="text">
        <string>ГОТОВО</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="logo">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/res/img/title.png);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
